Coroutine.ttslua

```diff
@@ -1,16 +1,28 @@
 ---@class ge_tts__Coroutine
 local Coroutine = {}
 
---- Yields from the current coroutine, which will later be resumed once a condition is met.
+--- Yields from the current coroutine, to be resumed once a condition is met or an optional timeout limit is reached.
```

Comment:
Not a fan of replacing:
> which will later

with:
> to

Just doesn't seem as specific about what's happening. Also a "timeout" is inherently a limit, you'd either say "time limit" or "timeout", however seems as there is a `timeout` parameter, just saying "timeout" is best.